llvm/tools/llvm-cxxfilt/llvm-cxxfilt.cpp

```diff
@@ -165,13 +165,9 @@ int llvm_cxxfilt_main(int argc, char **argv, const llvm::ToolContext &) {
     return 0;
   }
 
-  // The default value depends on the default triple. Mach-O has symbols
-  // prefixed with "_", so strip by default.
   if (opt::Arg *A =
           Args.getLastArg(OPT_strip_underscore, OPT_no_strip_underscore))
     StripUnderscore = A->getOption().matches(OPT_strip_underscore);
```

Comment:
Now this can be simplified without if statement. 
`StripUnderscore = Args.hasFlag(OPT_strip_underscore, OPT_no_strip_underscore, false)`

Reply:
nice, thanks!